doc/io.rst

```diff
@@ -79,28 +77,36 @@ require external libraries and dicts can easily be pickled, or converted to
 json, or geojson. All the values are converted to lists, so dicts might
 be quite large.
 
+.. _io.netcdf:
+
 netCDF
 ------
 
-Currently, the only disk based serialization format that xarray directly supports
-is `netCDF`__. netCDF is a file format for fully self-described datasets that
-is widely used in the geosciences and supported on almost all platforms. We use
-netCDF because xarray was based on the netCDF data model, so netCDF files on disk
+The recommended way to store xarray objects is `netCDF`__.
+netCDF is a file format for fully self-described datasets that
+is widely used and supported on almost all platforms. xarray is based on the
```

Comment:
I agree with the sentiment here, but I think it might still be a good idea to mention at least that netCDF "originated in the geosciences".